esp/esp/program/modules/handlers/commmodule.py

```diff
@@ -124,9 +125,13 @@ def commprev(self, request, tl, one, two, module, extra, prog):
 
         contextdict = {'user'   : ActionHandler(firstuser, firstuser),
                        'program': ActionHandler(self.program, firstuser),
-                       'request': ActionHandler(MessageRequest(), firstuser)}
+                       'request': ActionHandler(MessageRequest(), firstuser),
+                       'EMAIL_HOST_SENDER': settings.EMAIL_HOST_SENDER}
 
-        renderedtext = Template(body).render(DjangoContext(contextdict))
+        # Use whichever template the user selected or the default (just an unsubscribe slug) if 'None'
```

Comment:
How does someone select which template to use? I'm not seeing where the admin would have the chance to select the minimal template.

Reply:
One doesn't... yet. This is a stepping stone to a selectable set. That comment is probably bad form, but my hope is to get another PR out before the next release that actually allows selection from multiple. I can change it if you think I should